### PR TITLE
PermissionRegistry: Error on unknown kind

### PR DESCRIPTION
**Pull request by gamab**
**What is this feature?**

This PR makes sure we don't start Grafana with a registry improperly populated.
If there is a new kind (ex: "ice-cream") we have to make sure the associated RBAC scope is populated.

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Enterprise counterpart: https://github.com/grafana/grafana-enterprise/pull/7004

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/services/accesscontrol/permreg/permreg.go [hunk 1]
```diff
@@ -101,42 +107,50 @@ func (pr *permissionRegistry) RegisterPluginScope(scope string) {
...
+	pr.logger.Debug("registered scope prefix", "kind", kind, "scope_prefix", kind+":")
+	pr.kindScopePrefix[kind] = kind + ":"
 }
 
-func (pr *permissionRegistry) RegisterPermission(action, scope string) {
-	if _, ok := pr.actionScopePrefixes[action]; !ok {
-		pr.actionScopePrefixes[action] = PrefixSet{}
+func (pr *permissionRegistry) RegisterPermission(action, scope string) error {
+	if _, ok := pr.actionScopePrefixes[action]; ok {
+		// action already registered
```
Comment (eleijonmarck) [inline pkg/services/accesscontrol/permreg/permreg.go:131]: love it

**Review by eleijonmarck — APPROVED** [review]
👍 
Small comment. 

Follow up, add documentation a link to where the defaultKinds list is (not sure where this documentation could be though).

**Review by alexanderzobnin — APPROVED** [review]

**Review by IevaVasiljeva — APPROVED** [review]

**Pull request merged**